themeable-variables.json

```diff
@@ -745,6 +745,9 @@
     "--terra-signature-background-color": "#fff",
     "--terra-signature-line-color": "#000"
   },
+  "SlidePanel": {
+    "--terra-slide-panel-panel-width-small": "320px"
```

Comment:
That's strange. Should this be checked in?

Reply:
I think these are regularly out of sync. When making changes on my local branch I often accidentally generate new variables within this file as I rebase on master.

Reply:
I believe adding IndexPageTemplate resulted in these changes, which is why I checked them in. They can be reverted if we aren't supposed to let this file be updated.

Reply:
More info on this issue can be found here: https://github.com/cerner/terra-dev-site/issues/54.